Slides/researchdesignform-slides.Rmd

```diff
@@ -282,7 +282,7 @@ A campaign where someone visits a home to talk with a family for 15 minutes to s
 - Helps us be concrete about the stages of research design by allowing us
   to represent them in code, which then allows us to simulate the stages of research design in order to understand the properties of the statistical estimators and tests that we use.
 
-- For more see (https://declaredesign.org/getting-started/)
+- For more see (https://declaredesign.org/getting-started)
```

Comment:
It is surprising : with /, the link does not work

Reply:
Interesting. Thanks so much for catching this!